cli/cmd/project/status.go

```diff
@@ -55,8 +55,9 @@ func StatusCmd(ch *cmdutil.Helper) *cobra.Command {
 			fmt.Printf("  Organization: %v\n", proj.Project.OrgName)
 			fmt.Printf("  Public: %v\n", proj.Project.Public)
 			fmt.Printf("  Github: %v\n", githubURL)
-			fmt.Printf("  Created: %s\n", proj.Project.CreatedOn.AsTime().Local().Format(time.RFC3339))
-			fmt.Printf("  Updated: %s\n", proj.Project.UpdatedOn.AsTime().Local().Format(time.RFC3339))
+			fmt.Printf("  Created by: %s\n", proj.Project.CreatedByUserId)
```

Comment:
This user ID will be meaningless to normal users though? It's only us internally who are able to lookup by user ID, but this command is an end-user command